.stan.toml

```diff
@@ -4,4 +4,9 @@ directory = "target/"
 [[check]]
 type = "Exclude"
 directory = "test/"
-id = "STAN-0208"
\ No newline at end of file
+id = "STAN-0208"
+
+[[check]]
+type = "Exclude"
+file = "src/Stan/Cabal.hs"
```

Comment:
Could you add the comment in this module explaining why it is important to leave `unsafeInterleave` in there?